### PR TITLE
[wip] refactor: moving commands into dedicated packages

### PR DESCRIPTION
**Pull request by hferentschik**
fixes issue #4241

### CODE REVIEW

File: pkg/jx/cmd/create/create_cluster_minikube.go [hunk 1]
```diff
@@ -43,6 +43,14 @@ type CreateClusterMinikubeFlags struct {
 	ClusterVersion      string
 }
 
+const (
+	MinikubeDefaultCpu = "3"
+
+	MinikubeDefaultDiskSize = "150GB"
```
Comment (ccojocar) [inline pkg/jx/cmd/create/create_cluster_minikube.go]: Do we need to export these constants?

File: pkg/jx/cmd/create/create_devpod.go [hunk 2]
```diff
@@ -814,7 +815,7 @@ func (o *CreateDevPodOptions) Run() error {
 	}
 
 	if o.Sync {
-		syncOptions := &SyncOptions{
+		syncOptions := &sync.SyncOptions{
```
Comment (ccojocar) [inline pkg/jx/cmd/create/create_devpod.go]: In another round of refactoring we can remove the command name from the option:

`sync.SyncOptions` => `sync.Options`

File: pkg/jx/cmd/cmd.go [hunk 3]
```diff
@@ -18,6 +18,23 @@ package cmd
 
 import (
 	"fmt"
+	"github.com/jenkins-x/jx/pkg/jx/cmd/cloudbees"
```
[the remark this answers is not in the document]
Comment (hferentschik) [reply pkg/jx/cmd/cmd.go]: done

File: pkg/jx/cmd/create/create_cluster_minikube.go (hunk 1 above)
Comment (hferentschik) [reply pkg/jx/cmd/create/create_cluster_minikube.go]: no, I make them private

**Pull request closed without merging**